src/lib/ScissorsMe.ts

```diff
@@ -0,0 +1,108 @@
+import { path } from "@ffmpeg-installer/ffmpeg";
+import ffmpeg, { FfmpegCommandOptions } from "fluent-ffmpeg";
+ffmpeg.setFfmpegPath(path);
+
+import { existsSync, mkdirSync, unlink } from "fs";
+import { EventEmitter } from 'events';
+import { Readable } from "stream";
+import ytdl from "ytdl-core";
+import { cut } from "mp3-cutter";
+
+interface IOptions {
+  url: string;
+  alias: string;
+  start?: Number;
+  end?: Number;
+}
+
+export class ScissorsMe {
+  private event: EventEmitter;
+  private options: IOptions;
+  private _audioPath: string = `${__dirname}/mp3`;
```

Comment:
Since we are using typescript now, couldn't we drop the low dash convention for private attributes/methods?